public/apps/configuration/panels/tenant-list/test/__snapshots__/tenant-list.test.tsx.snap

```diff
@@ -82,6 +71,14 @@ exports[`Tenant list Action menu click Duplicate click 1`] = `
             >
               Create visualizations
             </EuiButtonEmpty>
+            <EuiButtonEmpty
+              disabled={true}
+              id="set as Global Default"
```

Comment:
Nit: avoid spaces in ids and class names. HTML does not allow spaces in `id` because it cannot be referenced by css. It looks like it works in React, but IMO spaces should be avoided.

Reply:
I can take this up later to fix this.